src/main/java/service/imp/ReaderServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package service.imp;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import service.ReaderService;
+
+public class ReaderServiceImpl implements ReaderService {
+    private List<String> dataFromFile = new ArrayList<>();
+
+    @Override
+    public List<String> read(String fileName) {
+        try {
+            dataFromFile = Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find data with this path:  " + fileName);
+        }
+        return dataFromFile;
+    }
+}
```

Comment:
The class `ReaderServiceImpl` should be located in the `service` package, not `service.imp`. The `.imp` (which I assume is short for `.impl`, implementation) package should only contain implementations of the interfaces, not the interfaces themselves.